src/App.js

```diff
@@ -1,11 +1,30 @@
 import React from 'react';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
+import './App.css';
 import Calculator from './components/calculator';
+import Header from './components/Header';
+import Home from './components/Home';
+import Quotes from './components/Quotes';
 
-// eslint-disable-next-line react/prefer-stateless-function
-class App extends React.Component {
+class App extends React.PureComponent {
   render() {
     return (
-      <Calculator />
+      <>
+        <Router>
+          <Header />
+          <Routes>
+            <Route path="/" element={<Home />} />
+            <Route path="/Header" element={<Header />} />
+            <Route path="/quote" element={<Quotes />} />
+          </Routes>
+        </Router>
+        <Calculator />
```

Comment:
Please ensure that your calculator component only shows when the user clicks on the calculator link 🤓 That way your app will meet the project requirements 🧐